src/nav/ModalNavBar.js

```diff
@@ -4,17 +4,15 @@ import { connect } from 'react-redux';
 import React, { PureComponent } from 'react';
 import { View } from 'react-native';
 
-import type { Dispatch, Context, GlobalState, LocalizableText } from '../types';
+import type { Dispatch, Context, LocalizableText } from '../types';
 import { NAVBAR_SIZE } from '../styles';
 import Label from '../common/Label';
-import { getCanGoBack } from '../selectors';
 import NavButton from './NavButton';
 import { navigateBack } from '../actions';
-import { connectPreserveOnBackOption } from '../utils/redux';
 
 type Props = {|
   dispatch: Dispatch,
-  canGoBack: boolean,
+  canGoBack?: boolean,
```

Comment:
This should not be optional because you always pass a value thanks to `defaultProps`

Reply:
Done 👍